pegasus/forms/census.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './form'
+
+class Census2017 < Form
+  def self.normalize(data)
+    result = {}
+
+    result[:email_s] = required data[:email_s]
+    result[:name_s] = data[:name_s]
+    result[:role_s] = data[:role_s]
```

Comment:
this one looks fine -- you don't need to strip `role_s` because it comes from a list of options which you defined. you may want to make it `required` though, if it is a required question.